src/sig_stfl/lms/external/hash.c

```diff
@@ -44,7 +46,8 @@ void hss_hash_ctx(void *result, int hash_type, union hash_context *ctx,
 #if ALLOW_VERBOSE
         if (hss_verbose) {
             printf( " ->" );
-            int i; for (i=0; i<32; i++) printf( " %02x", ((unsigned char *)result)[i] ); printf( "\n" );
+            int i; for (i=0; i<32; i++) printf( " %02x", ((unsigned char *)result)[i] ); printf( "
```

Comment:
Did something go wrong with the `\n` symbols here?

Reply:
Yep, will fix.